lib/src/components/StickyNotes/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { forwardRef } from 'react'
+
+import { CloseButton } from '@/components/CloseButton'
+import { classNames } from '@/utils'
+
+import { Button } from './components/Button'
+import { Title } from './components/Title'
+import styles from './sticky-notes.module.scss'
+import type { StickyNotesProps, StickyNotesTitleProps } from './types'
+
+const cx = classNames(styles)
+
+const StickyNotesComponent = forwardRef<HTMLDivElement, StickyNotesProps>(
+  ({ children, className, handleClose, shape = 'square', variant = 'brand' }, ref) => {
+    const hasCloseButton = Boolean(handleClose)
+
+    const cloneElements = () =>
+      React.Children.map(children, child => {
+        if (React.isValidElement<StickyNotesTitleProps>(child) && child.type === Title) {
+          return React.cloneElement(child, {
+            variant,
+          })
+        }
+
+        return child
+      })
```

Comment:
Can this be done with CSS instead ?

Reply:
From my understanding, the consuming app is responsible for hiding the sticky note. That's why you HAVE to pass a close function